tests/test_measurements.py

```diff
@@ -29,6 +29,19 @@
 if not ld._CPP_BINARY_AVAILABLE:
     pytest.skip("No binary module found. Skipping.", allow_module_level=True)
 
+if device_name == "lightning.qubit":
+    import pennylane_lightning.lightning_qubit_ops as lightning_ops
+elif device_name == "lightning.kokkos":
+    import pennylane_lightning.lightning_kokkos_ops as lightning_ops
+elif device_name == "lightning.gpu":
+    import pennylane_lightning.lightning_gpu_ops as lightning_ops
```

Comment:
It may be worth integrating this into our contest file. [Here](https://github.com/PennyLaneAI/pennylane-lightning/blob/master/tests/conftest.py#L120-L126)